scripts/query.sh

```
@@ -0,0 +1,91 @@
+
+send_curl_query(){
+    local query="$1"
+    
+    # Get fresh OCM token for this query
+    if ! get_ocm_token; then
+        echo "Failed to get OCM token for query"
+        return 1
+    fi
+
+    # Make the curl request and capture the response
+    local response=$(curl --silent \
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)